app/components/progress/index.js

```diff
@@ -12,8 +12,8 @@ const styles = StyleSheet.create({
   progress: {
     alignItems: 'center',
     flexDirection: 'row',
-    padding: 10,
-    color: '#00c8dd',
+    // padding: 10,
+    color: '#005fa3',
```

Comment:
I have been saving colors to `app/themes/colors.js` and importing from there. Not the best theming system but better than nothing...